spring-integration-file/src/test/java/org/springframework/integration/file/config/FileOutboundChannelAdapterParserTests.java

```diff
@@ -175,6 +184,15 @@ public void adapterWithDirectoryExpression() {
 	}
 
 	@Test
+	public void adapterWithAppendNewLine() {
+		DirectFieldAccessor adapterAccessor = new DirectFieldAccessor(adapterWithAppendNewLine);
+			FileWritingMessageHandler handler = (FileWritingMessageHandler)
+				adapterAccessor.getPropertyValue("handler");
+		DirectFieldAccessor handlerAccessor = new DirectFieldAccessor(handler);
+		assertEquals(Boolean.TRUE, handlerAccessor.getPropertyValue("appendNewLine"));
```

Comment:
For the future too. This can be simplified with:
````java
assertEquals(Boolean.TRUE, TestUtils.getPropertyValue(this.adapterWithAppendNewLine, "handler.appendNewLine"));
````
There is just need to study Framework! :smile: And get a gain from our experience